### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by giulianobernardi**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -57,9 +62,104 @@ void variableDeclarations()
...
+    ignoreUnused(uChar03);
+    ignoreUnused(float01);
+    ignoreUnused(float02);
+    ignoreUnused(float03);
+    ignoreUnused(double01);
+    ignoreUnused(double02);
+    ignoreUnused(double03);
+    ignoreUnused(longDouble01);
+    ignoreUnused(longDouble02);
+    ignoreUnused(longDouble03);
```
Comment (matkatmusic) [inline main.cpp]: You could have passed all of them at the same time to `ignoreUnused()`.  The example function below demonstrates this.

File: main.cpp [hunk 2]
```diff
@@ -74,67 +174,127 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
     
     //8)
+    startCounter(1);
     
     //9)
+    stopCounting(true);
     
     //10)
+    playAudioForNSeconds(10);
+
```
Comment (matkatmusic) [inline main.cpp]: I would like you to store the returned value from any function that returns a value into a local variable. 
At the end of main(), before the "good to go", pass all of these local variables to a single call of `ignoreUnused()`

- [x] please revise according to instructions above

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (giulianobernardi) [reply main.cpp]: I made the changes.

File: main.cpp (hunk 1 above)
Comment (giulianobernardi) [reply main.cpp]: OK

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**